TechnicoBackend/Repositories/RepairRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using TechnicoBackend.Data;
+using TechnicoBackend.Models;
+
+namespace TechnicoBackend.Repositories
+{
+    public class RepairRepository
```

Comment:
Consider using an interface IRepairRepository from which the current class would be inherited.